packages/catalog-server/src/test/lib/catalog_test.ts

```diff
@@ -49,12 +53,16 @@ test('Gets package version data from imported package', async () => {
   );
 
   assert.ok(result);
+  assert.equal(result.status, VersionStatus.READY);
+
+  const packageVersion = result as ReadablePackageVersion;
```

Comment:
If this was written:

```typescript
if (result.status !== VersionStatus.READY) {
  throw new Error('...');
}
```

would this `as` assertion be unnecessary?

Reply:
Yeah... I just wish we could get uvu asserts to also be type guards.